frontend/src/modules/Survey/Components/StepCourse.tsx

```diff
@@ -31,15 +35,19 @@ export const StepCourse = ({
     }
   }
 
-  const placeholder = courses.length === 0
-    ? 'ABC 1100'
-    : '+ Add another course...'
+  const placeholder =
+    courses.length === 0 ? 'ABC 1100' : '+ Add another course...'
 
   return (
     <StyledContainer>
       <StyledQuestionText>
         What course(s) would you like to find study partners for?
       </StyledQuestionText>
+      <StyledWarningText>
+        * Please do not submit the cross-listed version of the same course more
+        than once
```

Comment:
Again, I would advocate for less strong wording like "You do not need to submit the cross-listed version of the same course more than once" because it's a strength that we don't need them to do it multiple times, not a flaw on our end that submitting the cross-listed version more than once would break something. I made a comment on the Figma about this just now.